### PR TITLE
New presets

### PR DESCRIPTION
**Pull request by Secret-chest**
Resolves #

### Changes

Added three new presets to some addons: Oceanic Dark to editor and website dark mode, and Material to custom block colors.

### Reason for changes

* The material block colors are easy to tell apart and look colorful without straining the eye. They go well with the oceanic dark one too!
* The oceanic dark theme is really popular. If you look up those colors (\#263238 and \#0\09688) on Google, it gives you many results, including https://www.color-hex.com/color/009688 and https://www.color-hex.com/color/263238, which show 14 and 8 favorites, respectively. Which means many liked those colors.

### Tests

No, i'm 100% sure it will work, it's just 3 presets

### Info

I would like to be credited by my GitHub username

### CODE REVIEW

File: addons/editor-theme3/addon.json [hunk 1]
```diff
@@ -231,6 +235,27 @@
...
+        "sensing-color": "#5CB1D6",
+        "operators-color": "#59C059",
+        "data-color": "#FF8C1A",
+        "data-lists-color": "#FF661A",
+        "custom-color": "#FF6680",
+        "Pen-color": "#0FBD8C",
+        "sa-color": "#29BEB8",
+        "input-color": "#202020",
+        "text": "colorOnBlack"
+      }
```
Comment (mxmou) [inline addons/editor-theme3/addon.json:258]: Are you sure that these are intended values? It's just a copy of the Black preset.

**Comment by mxmou** [comment]
Is the "oceanic" theme used in any Scratch mod? I don't think themes unrelated to Scratch should be included with Scratch Addons - something like #2739 that would allow sharing custom themes would be better.

File: addons/dark-www/addon.json [hunk 2]
```diff
@@ -1943,6 +1947,26 @@
...
+        "button": "#009688ff",
+        "darkBanners": "off",
+        "darkForumCode": true,
+        "darkScrollbars": true,
+        "footer": "#37474fff",
+        "gray": "#37474fff",
+        "input": "#455a64ff",
+        "link": "#009688ff",
+        "navbar": "#009688ff",
+        "page": "#263238ff",
```
Comment (Samq64) [inline addons/dark-www/addon.json:1967]:
```suggestion
        "page": "#263238ff"
```

File: addons/editor-dark-mode/addon.json [hunk 3]
```diff
@@ -1093,6 +1097,34 @@
         "darkScrollbars": false
       }
     },
+    {
+      "name": "Oceanic Dark",
+      "id": "oceanic",
+      "description": "A material theme based on the very popular blue gray (#263238) and teal (#009688) combination",
+      "values": {
```
Comment (Samq64) [inline addons/editor-dark-mode/addon.json:1104]: The values of both Oceanic Dark presets should use the same order as the rest of the presets to make them easier to read.

File: addons/dark-www/addon.json [hunk 4]
```diff
@@ -11,6 +11,10 @@
       "name": "Ucrash (Dark WWW)",
       "link": "https://github.com/gducrash"
     }
```
Comment (Samq64) [inline addons/dark-www/addon.json:13]:
```suggestion
    },
```

File: addons/editor-dark-mode/addon.json [hunk 5]
```diff
@@ -18,6 +18,10 @@
       "name": "GarboMuffin (TurboWarp Dark)",
       "link": "https://github.com/GarboMuffin"
     }
```
Comment (Samq64) [inline addons/editor-dark-mode/addon.json:20]:
```suggestion
    },
```

File: addons/editor-theme3/addon.json (hunk 1 above)
Comment (Secret-chest) [reply addons/editor-theme3/addon.json:258]: No...

Comment (Secret-chest) [reply addons/editor-theme3/addon.json]: Fixed

**Comment by WorldLanguages** [comment]
I'm not 100% sure we will decide in favor of adding user submitted presets until we have a real theme library.

**Comment by Secret-chest** [comment]
> Is the "oceanic" theme used in any Scratch mod? I don't think themes unrelated to Scratch should be included with Scratch Addons - something like #2739 that would allow sharing custom themes would be better.

Are the original 4 onei in Scratch mods?

**Comment by Secret-chest** [comment]
> I'm not 100% sure we will decide in favor of adding user submitted presets until we have a real theme library. 

@WorldLanguages But it's a really popular dark color scheme.

**Comment by mxmou** [comment]
> > Is the "oceanic" theme used in any Scratch mod? I don't think themes unrelated to Scratch should be included with Scratch Addons - something like #2739 that would allow sharing custom themes would be better.
> 
> Are the original 4 ones in Scratch mods?

They're from userscripts and userstyles made specifically for Scratch.

**Comment by Secret-chest** [comment]
> > > Is the "oceanic" theme used in any Scratch mod? I don't think themes unrelated to Scratch should be included with Scratch Addons - something like #2739 that would allow sharing custom themes would be better.
> > 
> > Are the original 4 ones in Scratch mods?
> 
> They're from userscripts and userstyles made specifically for Scratch.

OK, but this is really popular and it would fit for Scratch. It's not that sort of a rainbow theme that would be posted in the presets section. Please add it.

**Comment by Secret-chest** [comment]
> 
> 
> Resolves #
> 
> ### Changes
> 
> 
> Added three new presets to some addons: Oceanic Dark to editor and website dark mode, and Material to custom block colors.
> 
> ### Reason for changes
> 
> * The material block colors are easy to tell apart and look colorful without straining the eye. They go well with the oceanic dark one too!
> * The oceanic dark theme is really popular. If you look up those colors (\#263238 and \#\009688) on Google, it gives you many results, including https://www.color-hex.com/color/009688 and https://www.color-hex.com/color/263238, which show 14 and 8 favorites, respectively. Which means many liked those colors.
> 
> ### Tests
> 
> No, i'm 100% sure it will work, it's just 3 presets
> 
> ### Info
> 
> I would like to be credited by my GitHub username

**Comment by Secret-chest** [comment]
> 
> 
> Resolves #
> 
> ### Changes
> 
> 
> Added three new presets to some addons: Oceanic Dark to editor and website dark mode, and Material to custom block colors.
> 
> ### Reason for changes
> 
> * The material block colors are easy to tell apart and look colorful without straining the eye. They go well with the oceanic dark one too!
> * The oceanic dark theme is really popular. If you look up those colors (\#263238 and \#009688) on Google, it gives you many results, including https://www.color-hex.com/color/009688 and https://www.color-hex.com/color/263238, which show 14 and 8 favorites, respectively. Which means many liked those colors.
> 
> ### Tests
> 
> No, i'm 100% sure it will work, it's just 3 presets
> 
> ### Info
> 
> I would like to be credited by my GitHub username

**Pull request closed without merging**